### DISCUSSION TITLE
reactiveTransportDBSFoam momentum formulation

### DISCUSSION THREAD

**Discussion by Mahsa-Hs** [GeoChemFoam/GeoChemFoam#43] [Q&A] [answered]
Hi,
In Equation (11) in the "Improved Volume-Of-Solid Formulations for Micro-Continuum Simulation of Mineral Dissolution at the Pore-Scale" article, the laplacian term is multiplied by (1/eps). However, in the UEqn.H file in reactiveTranspportDBSFoam solver, I cannot see any (1/eps) for this term. Can you please explain it?

**Comment by GeoChemFoam** [comment, marked as answer]
Hi Mahsa,

If you looked back at versions we divide by epsilon and this is what we used in the simulations associated with this paper. However, whether to divide by epsilon is an active research question that we are investigating as to the best way to do this and we haven’t decided what the right answer is. You can change the source code and recompile if you prefer the other method.

Hope this helps

Hannah

From: Mahsa-Hs ***@***.***>
Date: Wednesday, 6 March 2024 at 10:11
To: GeoChemFoam/GeoChemFoam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoChemFoam/GeoChemFoam] reactiveTransportDBSFoam momentum formulation (Discussion #43)
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hi,
In Equation (11) in the "Improved Volume-Of-Solid Formulations for Micro-Continuum Simulation of Mineral Dissolution at the Pore-Scale" article, the laplacian term is multiplied by (1/eps). However, in the UEqn.H file in reactiveTranspportDBSFoam solver, I cannot see any (1/eps) for this term. Can you please explain it?

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/43>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFT5LFJJXALOSUKIDF3YW3TTPAVCNFSM6AAAAABEIX33YCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGMZTANRRG4>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.
